pkg/internal/checkup/executor/trex-console.go

```diff
@@ -12,16 +12,16 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-type trexConsole struct {
+type trexClient struct {
 	podClient            podExecuteClient
 	namespace            string
 	name                 string
 	containerName        string
 	verbosePrintsEnabled bool
 }
 
-func NewTrexConsole(client podExecuteClient, namespace, name, containerName string, verbosePrintsEnabled bool) trexConsole {
-	return trexConsole{
+func NewTrexClient(client podExecuteClient, namespace, name, containerName string, verbosePrintsEnabled bool) trexClient {
```

Comment:
nit:
```suggestion
func New(client podExecuteClient, namespace, name, containerName string, verbosePrintsEnabled bool) trexClient {
```

It could make the call less verbose:

```go
trexclient.New(....)
```

Reply:
Or, if the package will be renamed to`trex`
`trex.NewClient()` could also be nice.

Reply:
DONE

Reply:
I changed it in a separate commit to avoid the complexity of it.